packages/ethereum-storage/test/ipfs-manager.test.ts

```diff
@@ -52,40 +45,43 @@ const contentLengthOnIpfs2 = 38;
 /* eslint-disable no-magic-numbers */
 describe('Ipfs manager', () => {
   beforeEach(() => {
-    ipfsManager = new IpfsManager(ipfsGatewayConnection, testErrorHandling);
+    ipfsManager = new IpfsManager({
+      ipfsGatewayConnection,
+      ipfsErrorHandling: testErrorHandling,
+    });
+  });
+
+  it('initializing ipfs-manager with default values should not throw any error', async () => {
+    expect(() => new IpfsManager()).not.toThrow();
   });
 
   it('allows to verify repository', async () => {
     await ipfsManager.getIpfsNodeId();
-
-    ipfsManager = new IpfsManager(invalidHostIpfsGatewayConnection, testErrorHandling);
-    await expect(ipfsManager.getIpfsNodeId()).rejects.toThrowError('getaddrinfo ENOTFOUND');
+    await expect(ipfsManager.getIpfsNodeId()).resolves.toMatchObject({ ID: /.+/ });
   });
 
   it('allows to get the bootstrap list', async () => {
     const bootstrapList = await ipfsManager.getBootstrapList();
     expect(bootstrapList).toHaveProperty('length');
   });
 
-  it('must throw if the ipfs node is not reachable when using getBootstrapList()', async () => {
-    ipfsManager = new IpfsManager(invalidHostIpfsGatewayConnection, testErrorHandling);
-    await expect(ipfsManager.getBootstrapList()).rejects.toThrowError('getaddrinfo ENOTFOUND');
-  });
-
   it('allows to add files to ipfs', async () => {
-    let hashReturned = await ipfsManager.add(content);
-    expect(hash).toBe(hashReturned);
+    const hashReturned1 = await ipfsManager.add(content);
+    expect(hashReturned1).toBe(hash);
 
-    hashReturned = await ipfsManager.add(content2);
-    expect(hash2).toBe(hashReturned);
+    const hashReturned2 = await ipfsManager.add(content2);
+    expect(hashReturned2).toBe(hash2);
   });
 
   it('allows to pin one file ipfs', async () => {
+    await ipfsManager.add(content);
     const pinnedHash = await ipfsManager.pin([hash]);
     expect(hash).toBe(pinnedHash[0]);
   });
 
   it('allows to pin multiple files to ipfs', async () => {
+    await ipfsManager.add(content);
+    await ipfsManager.add(content2);
```

Comment:
Those added lines are only to make tests independent of each other (to be able to run those tests alone)